src/pages/home/HeaderView.js

```diff
@@ -371,7 +407,6 @@ export default memo(
         },
         policy: {
             key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '0'}`,
-            selector: (policy) => _.pick(policy, ['name', 'avatar', 'pendingAction']),
```

Comment:
Could you explain why we need to make this change?

Reply:
@jjcoffee We need to use many other properties on the policy object like `role`, `description`. I don't see a selector like this being used anywhere else for policy because we are querying only one policy and the collection policies, I don't think it will hurt to not use the selector and use the whole policy object.